src/scenes/characters/character.gd

```diff
@@ -33,6 +33,21 @@ func check_quest_status() -> void:
 
 func dialog_ended(_timeline_name) -> void:
     GameState.set_is_in_dialogue(false);
+    var timeline_name_length = _timeline_name.length();
+    var timeline_number = int(_timeline_name[timeline_name_length - 1]);
+    if (timeline_number == 2):
+        return;
```

Comment:
It would be useful to have these special timeline numbers as named constants. What does the value of 2 represent and why is it a special case?

Reply:
When `timeline_number` equals 2, the "Quest Start" dialog timeline occurs which doesn't provide any additional hints for the culprit. Therefore, this early return is to avoid providing hints for this particular timeline.

The timelines should follow the following stages:
1. Hint 1 is provided
2. Hint 2 is provided
3. Quest Start, no hints provided
4. Quest End, hint 3 is provided

Reply:
The timelines have been updated to follow the above stages.

Reply:
I've added a comment in the `if` statement for code readability.